addons/web/static/src/views/calendar/date_utils.js

```diff
@@ -0,0 +1,20 @@
+/** @odoo-module **/
+
+import {
+    deserializeDate,
+    deserializeDateTime,
+    serializeDate,
+    serializeDateTime,
+} from "@web/core/l10n/dates";
+
+// export function calculateWeekNumber(date) {
+//     //WOWL: USED ONLY FOR THE DATEPICKER (NEEDED ???)
+//     return luxon.DateTime.fromJSDate(date).weekNumber;
+// }
+
+export function deserialize(value, type) {
+    return type === "date" ? deserializeDate(value) : deserializeDateTime(value);
+}
+export function serialize(value, type) {
+    return type === "date" ? serializeDate(value) : serializeDateTime(value);
+}
```

Comment:
not sure about this file. I would either get rid of those two helpers, or move them directly into dates